src/app/components/turnos/gestor-agendas/operaciones-agenda/clonar-agenda.ts

```diff
@@ -213,9 +213,15 @@ export class ClonarAgendaComponent implements OnInit {
             if (moment(dia.fecha).isSame(moment(agenda.horaInicio), 'day')) {
                 if (agenda.profesionales && agenda.profesionales.length > 0) {
                     if (agenda.profesionales.map(elem => { return elem.id; }).some
-                        (v => { return this.agenda.profesionales.map(elem => { return elem.id; }).includes(v); })) {
+                        (v => {
+                            if (this.agenda.profesionales) {
```

Comment:
Cambié de lugar esta validación porque no tenía sentido acá dentro. Si `this.agenda.profesionales`es falso, siempre va a devolver falso, entonces se estaría recorriendo el listado de agendas al vicio, por lo menos por parte de profesionales (puede tener conflicto por espacio físico).
Le agregué también una validación para que se ejecute el método solo cuando `this.agenda`  tiene profesional y/o espacio físico. Si no tiene ninguna, de entrada sabemos que no va a tener conflicto